components/camel-sql/src/main/java/org/apache/camel/component/sql/stored/template/generated/SSPTParser.java

```diff
@@ -1,31 +1,11 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License.  You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
 /* Generated By:JavaCC: Do not edit this line. SSPTParser.java */
 package org.apache.camel.component.sql.stored.template.generated;
 
-import org.apache.camel.component.sql.stored.template.ast.InOutParameter;
-import org.apache.camel.component.sql.stored.template.ast.InParameter;
-import org.apache.camel.component.sql.stored.template.ast.OutParameter;
-import org.apache.camel.component.sql.stored.template.ast.ParseHelper;
-import org.apache.camel.component.sql.stored.template.ast.Template;
-import org.apache.camel.spi.ClassResolver;
-
 import java.io.Reader;
 
+import org.apache.camel.spi.ClassResolver;
+import org.apache.camel.component.sql.stored.template.ast.*;
+
 public class SSPTParser implements SSPTParserConstants {
```

Comment:
Not sure about this change (what caused it), but we seem to have lost the ASF license header and I think Checkstyle won't allow star imports. Perhaps just revert changes to this file?

Reply:
Hello Mr.Zoran I have no idea what caused to this, I just ran ./mvnw -Pfastinstall install.